scripts/check_md5sums.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ] || ! { [ "$1" = "zson" ] || [ "$1" = "bzson" ]; }; then
```

Comment:
Nit: Modern bash allows numerical expressions, like `(($# != 1))`. It's a big pain to be sh-compliant, and it doesn't look like we're trying to be. My suggestion would be to use the modern form.

Nit2: I think you can also just use `[[ $1 =~ ^b?zson$ ]]` for another modern touch.

Reply:
Thanks for the tips. I've made these changes and they test out fine.

Reply:
Sorry to sound like a nitter. It seems like you didn't push the changes discussed above. It sounds like you had decided to, so I wanted to point that out.

Reply:
Gah! Thanks for catching that. Stupid mistake. I was working in a scratch directory at one point and I think I made the change there by mistake. Fixed now.